edge/pkg/edged/volume/csi/csi_client.go

```diff
@@ -217,7 +217,7 @@ func (c *csiDriverClient) NodeGetInfo(ctx context.Context) (
 		var getNodeInfoError error
 		if c.nodeV1ClientCreator != nil {
 			nodeID, maxVolumePerNode, accessibleTopology, getNodeInfoError = c.nodeGetInfoV1(ctx)
-		} else if c.nodeV1ClientCreator != nil {
+		} else {
```

Comment:
`c.nodeGetInfoV1` will assign values to `c.nodeV1ClientCreator`, so the second `if` statement may be different from the first `if`